spring-cloud-starter-task-composedtaskrunner/README.adoc

```diff
@@ -0,0 +1,134 @@
+//tag::ref-doc[]
+= Composed Task Runner
+
+A task that executes a tasks in a directed graph as specified by a DSL that is
+passed in via the `--graph` command line argument.
+
+== Overview
+The Composed Task Runner parses the graph DSL and for each node in the graph it
+will execute a restful call against a specified http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/[Spring Cloud Data Flow]
+instance to launch the associated task definition.  For each task definition that is executed the
+Composed Task Runner will poll the database to verify that the task completed.
+Once complete the Composed Task Runner will either continue to the next task in
+the graph or fail based on how the DSL specified the sequence of tasks should
+be executed.
+
+== Graph DSL
+
+The Graph DSL is comprised of Task Definitions that have been defined within
+the Spring Cloud Data Flow server referenced by the data-flow-uri
+(default: http://localhost:9393).
+These definitions can be placed into a derived graph based on a DSL through
+the use of sequences, transitions, splits, or a combination therein.
+
+=== Sequences
+The Composed Task Runner supports the ability to traverse sequences of task
+definitions.  This is represented by a task definition name followed by the
+`&&` symbol then the next task definition to be launched.
+For example if we have tasks AAA, BBB and CCC to be launched in sequence it
+will look  like this:
+```
+AAA && BBB && CCC
+```
+You can execute the same task multiple times in a sequence.  For example:
+```
+AAA && AAA && AAA
+```
+If an ExitStatus 'FAILED' is returned in a sequence the Composed Task
```

Comment:
moved contents around so that batch would be described before traversing the graph